packages/plugin-dev/src/browser/hosted-plugin-frontend-contribution.ts

```diff
@@ -0,0 +1,45 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { CommandRegistry, CommandContribution } from '@theia/core/lib/common';
+import { HostedPluginManagerClient, HostedPluginCommands } from './hosted-plugin-manager-client';
+
+@injectable()
+export class HostedPluginFrontendContribution implements CommandContribution {
```

Comment:
'Hosted' name will remain ? because it's strange to have module named `plugin-dev` while classes are about `hosted` ?

Reply:
'Hosted' prefix will remain in 'plugin-dev' extension. I will remove all 'Hosted' prefixes from 'plugin-ext' extension in future pull's. 
I just keep all existing names.

Reply:
+100 on this point: for example "HostedPluginProcess" has nothing to do with this module, but uses the same name. Getting the naming correct here will greatly help with thinking about the concepts.